website/src/app/room/error-popup.html

```diff
@@ -0,0 +1,5 @@
+<h2 mat-dialog-title>Error Issue</h2>
+<div mat-dialong-content></div>
```

Comment:
It doesn't look like we are showing any error message here?  Can we add an error message that is either static or passed in to tell the user what went wrong?